test/conftest.py

```diff
@@ -53,5 +51,7 @@ def subprocess_popen(*args):
 
 
 def get_seed(endpoint):
+    retries = Retry(connect=5, total=None, backoff_factor=0.1)
```

Comment:
So instead of waiting for the server to start, we'll just retry the request if the server hasn't started? Neat!